### DISCUSSION TITLE
How to enable HTTPS using Airflow chart

### DISCUSSION THREAD

**Discussion by rafaelwsan** [apache/airflow#18688] [General]
Hi Folks,

I'm trying to enable HTTPS connection in my Airflow environment by Airflow official chart, but haven't found documentation or examples to help.

This issue (https://github.com/apache/airflow/issues/16010) treats the same difficulty, but in this case, the environment was AWS, but in my scenario, we use Azure (AKS), then I think it's not applicable.

Some environment info:
- kubernetes version: v1.20.9
- chart version: airflow-1.1.0
- airflow version: 2.1.2

Are there any quick tips or examples you could share with us?

**Comment by potiuk** [comment]
https://airflow.apache.org/docs/helm-chart/stable/production-guide.html#accessing-the-airflow-ui - you have to setup either Ingress or Load Balancer as described in the Production Guide.
